corehq/apps/user_importer/helpers.py

```diff
@@ -32,7 +32,11 @@ def __init__(self, domain, user, is_new_user, changed_by_user, changed_via):
         self.changed_by_user = changed_by_user
         self.changed_via = changed_via
 
-        self.original_user_doc = self.user.to_json()
+        if not is_new_user:
+            self.original_user_doc = self.user.to_json()
+        else:
```

Comment:
Isn't this branch unnecessary?

Reply:
https://github.com/dimagi/commcare-hq/commit/07746469f1d10d6907dee5ca1d547c6f3eb71cb0#diff-2f11ee2db6cd89244e3a42d3ed2a050b2bb42fde7eb86234a123855a6d75bcc2R120

Ya.